packages/manager/modules/pci/src/projects/project/storages/object-storage/users/add/user-add/user-add.controller.js

```diff
@@ -24,19 +24,20 @@ export default class PciUsersAddController {
     this.isLoading = false;
     this.addExistingUser = 'addExistingUser';
     this.createNewUser = 'createNewUser';
-
-    this.allUserList = this.allUserList.map((userList) => ({
-      ...userList,
-      asCredentials: this.usersCredentials.find(
-        (credential) => credential.userId === userList.id,
-      ).asCredentials
-        ? this.$translate.instant(
-            'pci_projects_project_users_add_as_credentials',
-          )
-        : this.$translate.instant(
-            'pci_projects_project_users_add_as_no_credentials',
-          ),
-    }));
+    this.allUserList = this.allUserList
+      .filter((allUserList) => allUserList)
+      .map((userList) => ({
+        ...userList,
+        asCredentials: this.usersCredentials.find(
+          (credential) => credential.userId === userList.id,
+        ).asCredentials
+          ? this.$translate.instant(
+              'pci_projects_project_users_add_as_credentials',
+            )
+          : this.$translate.instant(
+              'pci_projects_project_users_add_as_no_credentials',
+            ),
```

Comment:
**nitpick:** variable name is misleading since in each iteration the reference is to `user` and not `userList`
```suggestion
      .filter((user) => user)
      .map((user) => ({
        ...user,
        asCredentials: this.usersCredentials.find(
          (credential) => credential.userId === user.id,
        ).asCredentials
          ? this.$translate.instant(
              'pci_projects_project_users_add_as_credentials',
            )
          : this.$translate.instant(
              'pci_projects_project_users_add_as_no_credentials',
            ),
```

Reply:
Okay